### PR TITLE
Skip unit tests in jaws deploy orb

### PR DESCRIPTION
**Pull request by DanMaly**
+ add gradle parameter for running e2e tests.

### CODE REVIEW

**Review by robturnerovo — APPROVED** [review]
👍

File: jaws-journey-deploy/build_project.sh [hunk 1]
```diff
@@ -1 +1,9 @@
-./gradlew :"<< parameters.serviceName >>":clean :"<< parameters.serviceName >>":buildNeeded -Pprofile=${PROFILE} -x integrationTest --full-stacktrace
+SKIP_UNIT_TESTS="<< parameters.skipUnitTests >>"
```
Comment (ChrisFord-Ovo) [inline jaws-journey-deploy/build_project.sh:1]: might be worth updating the readme

**Review by ChrisFord-Ovo — CHANGES_REQUESTED** [review]
update readme to reflect change

**Review by ChrisFord-Ovo — APPROVED** [review]
lgtm

**Pull request merged**